### DISCUSSION TITLE
Aiguillage - affichage d'une réponse de l'aiguillage dans le formulaire suivant

### DISCUSSION THREAD

**Discussion by florentarneodo** [MTESSDev/FRW#124] [Annonces]
Bonjour,

Pouvez-vous nous donner la recette pour réutiliser la réponse d'une question de l'aiguillage pour l'afficher sur le formulaire suivant ou même pour faire des conditions avec éventuellement.

L'objectif est de ne pas demander 2 fois la même chose au client. Est-ce que toutes les réponses suivent automatiquement ou il y a une manipulation quelconque?

Merci et bonne journée.

**Comment by LarcM7** [comment]
Bonjour, 

Les questions et réponses suivent automatiquement dans le formulaire suivant. Vous n'avez qu'à prévoir un champ avec le même nom dans le formulaire suivant et la réponse fournie dans le premier formulaire sera affichée.

Est-ce que cela répond à votre question?

**Reply by florentarneodo** [reply]
oui merci!
